packages/components/src/color-picker/hex-input.tsx

```diff
@@ -24,6 +24,10 @@ interface HexInputProps {
 }
 
 export const HexInput = ( { color, onChange, enableAlpha }: HexInputProps ) => {
+	const handleChange = ( nextValue ) => {
+		const hexValue = ( '#' + nextValue ).replace( '##', '#' );
```

Comment:
This is more minor, but I'd recommend checking if `#` already is in nextValue, or stripping all `#` before adding a prefix.

Reply:
Thanks that looks good to me!